pkg/kubelet/prober/prober.go

```diff
@@ -49,6 +50,8 @@ type prober struct {
 	runner kubecontainer.CommandRunner
 
 	recorder record.EventRecorder
+
+	httpProbeReqCache sync.Map
```

Comment:
This should be moved to the http probe code, I believe.

Reply:
sure